Without this patch the printf invocation takes more argument that it has
format strings, since the format_qos() output string is printed in the
if statement following the HOST_PRINTF invocation, I believe this to be
an accidental error made in the following commit:

	https://github.com/octo/liboping/commit/66464b61f8ee756dcfc0081944f4367da2b1a6ab

See https://github.com/octo/liboping/pull/60

diff -upr liboping-1.10.0.orig/src/oping.c liboping-1.10.0/src/oping.c
--- liboping-1.10.0.orig/src/oping.c	2021-10-24 13:13:36.850533766 +0200
+++ liboping-1.10.0/src/oping.c	2021-10-24 13:13:49.790711547 +0200
@@ -1600,8 +1600,7 @@ static void update_host_hook (pingobj_it
 
 			HOST_PRINTF ("%zu bytes from %s (%s): icmp_seq=%u ttl=%i ",
 					data_len, context->host, context->addr,
-					sequence, recv_ttl,
-					format_qos (recv_qos, recv_qos_str, sizeof (recv_qos_str)));
+					sequence, recv_ttl);
 			if ((recv_qos != 0) || (opt_send_qos != 0))
 			{
 				HOST_PRINTF ("qos=%s ",
